CMakeLists.txt

```diff
@@ -335,6 +335,9 @@ if(HAVE_CLOCK_GETTIME OR HAVE_CLOCK_GETTIME_LIBRT)
     tuklib_add_definition_if(ALL HAVE_CLOCK_MONOTONIC)
 endif()
 
+check_symbol_exists(getrlimit sys/resource.h HAVE_GETRLIMIT)
+check_symbol_exists(RLIMIT_DATA sys/resource.h HAVE_DECL_RLIMIT_DATA)
+check_symbol_exists(RLIM_INFINITY sys/resource.h HAVE_DECL_RLIM_INFINITY)
 
```

Comment:
These set CMake variables like `HAVE_GETRLIMIT`. To create a C preprocessor macro, something like

```
if(HAVE_GETRLIMIT)
    add_compile_definitions(HAVE_GETRLIMIT)
endif()
```

is needed in addition. Or in this project, one can use a shorthand:

```
tuklib_add_definition_if(ALL HAVE_GETRLIMIT)
```